blueprints/maximum-plaid/index.js

```diff
@@ -0,0 +1,24 @@
+/*jshint node:true*/
+module.exports = {
+  description: 'Installation blueprint for maximum plaid',
+  normalizeEntityName: function () {},
+  beforeInstall: function() {
+    return this.addAddonsToProject({
+      packages: [
+        'ember-d3@~0.2.0',
```

Comment:
Ah yes, it's a poorly named library now. It is just a shim for D3 v4 code from NPM. I was originally going to build it into a full fledged vis library, which instead became maximum-plaid.